meson.build

```diff
@@ -262,7 +262,12 @@ graphene_simd = []
 
 # SSE intrinsics
 sse2_cflags = []
-if get_option('sse2')
+try_sse2 = get_option('sse2').enabled()
+if get_option('sse2').auto() and host_machine.cpu_family() == 'x86_64'
```

Comment:
I don't really want to have Meson test for this; the `sse_prog` below should be modified to fail is the architecture is not 64 bit.

Reply:
It's actually easier, just remove some conditions in `#if`.